### DISCUSSION TITLE
Mercado Livre endpoints e métodos

### DISCUSSION THREAD

**Discussion by fabiozatta** [rodurma/forum-meli#22] [Geral]
Boa tarde estou começando a integração com o ML mas a documentação da api é horrível. 

Alguem sabe se existe algo mais definido com os endpoints e métodos aceitos, put, post , delete,

**Comment by rodurma** [comment]
Seria ótimo se eles disponibilizassem um swagger com todos os endpoints reunidos. Quem sabe a gente não inicia isso aqui de forma não oficial.

**Comment by rodurma** [comment]
Fiz um exemplo de documentação usando swagger. 

Para ver alguns endpoints acesse https://rodurma.github.io/forum-meli/

Fiquem a vontade em fazer PR com outros endpoints no arquivo `swagger.json`
